great_expectations/rule_based_profiler/data_assistant/onboarding_data_assistant.py

```diff
@@ -927,7 +927,7 @@ def _build_categorical_columns_rule() -> Rule:
                 include_semantic_types=None,
                 exclude_semantic_types=None,
                 allowed_semantic_types_passthrough=None,
-                cardinality_limit_mode=CardinalityLimitMode.FEW,
+                cardinality_limit_mode=f"{VARIABLES_KEY}cardinality_limit_mode",
```

Comment:
Could you explain what is going on here? What is `VARIABLES_KEY` and why are changing from an enum to a string? Is the "FEW" comment above still valid?

Reply:
@billdirks This is part of the somewhat general profiler syntax -- happy to discuss separately.